migration/01_create_passwordreset.py

```diff
@@ -0,0 +1,30 @@
+import re
+import argparse
+from sqlalchemy.sql import text
+
+try:
+    from library import app, db
+    from models import PasswordReset
+except ImportError:
+    app, db, PasswordReset = None, None, None
+
+def create_passwordreset():
+    """
+    Create the passwordreset table in the database.
+    """
+    with app.app_context():
+        conn = db.session.connection()
+        table = PasswordReset.metadata.tables['passwordreset']
```

Comment:
Just found I could have used `PasswordReset.__tablename__` instead of `'passwordreset'`.